### PR TITLE
set-ios-product-bundle-identifier 1.1.1

### PR DESCRIPTION
**Pull request by emenegro**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by emenegro** [comment]
This new version adds `description` fields to the inputs, to make it clearer in the web frontend.

File: steps/set-ios-product-bundle-identifier/1.1.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+title: Replaces Xcode's project «PRODUCT_BUNDLE_IDENTIFIER
```
Comment (godrei) [inline steps/set-ios-product-bundle-identifier/1.1.1/step.yml]: is the `«` character required here?

Comment (emenegro) [inline steps/set-ios-product-bundle-identifier/1.1.1/step.yml]: My mistake, sorry!

**Comment by godrei** [comment]
Hi @emenegro ,
thanks for the changes!

Could you please re-share the step.

You made the changes in the forked step library, but you did not changed your step's title in the step's repository. If we would accept this pr, the step's title would be ok in the steplib, but when you create a new version from the step, will would see the old step's title (`«PRODUCT_BUNDLE_IDENTIFIER`).

**Comment by emenegro** [comment]
Done @godrei, I forgot it.

**Comment by godrei** [comment]
@emenegro , for a proper re-share please:
-  close this pr
- delete the `set-ios-product-bundle-identifier-1.1.1` branch in your steplib fork
- share the step again

**Pull request closed without merging**